templates/CRM/Event/Form/Participant.tpl

```diff
@@ -85,12 +87,7 @@
           {/if}
           <tr class="crm-participant-form-block-event_id">
             <td class="label">{$form.event_id.label}</td>
-            <td class="view-value">
-              {$form.event_id.html}
-              {if $is_test}
-                {ts}(test){/ts}
-              {/if}
-            </td>
+            <td class="view-value">{$form.event_id.html}</td>
```

Comment:
is_test was nothing.

Reply:
@larssandergreen was it supposed to be template (don't think one could register to event template though)

Reply:
Unless I'm remembering when event templates was introduced wrong, I think this predates event templates.